client/web/src/site-admin/SiteAdminUpdatesPage.tsx

```diff
@@ -1,23 +1,213 @@
+                            />
+                        )}
+                        {!data.site.updateCheck.errorMessage && (
+                            <small>
+                                {data.site.updateCheck.updateVersionAvailable ? (
+                                    <Link to="https://about.sourcegraph.com">
+                                        Update available to version {data.site.updateCheck.updateVersionAvailable}{' '}
+                                        <Icon aria-hidden={true} svgPath={mdiOpenInNew} />
+                                    </Link>
+                                ) : (
+                                    <span>
+                                        <Icon
+                                            aria-hidden={true}
+                                            className="text-success mr-1"
+                                            svgPath={mdiCheckCircle}
+                                        />
+                                        Up to date
+                                    </span>
+                                )}
+                                <span className={classNames('text-muted pl-2 ml-2', styles.lastChecked)}>
+                                    {data.site.updateCheck.checkedAt
+                                        ? `Last checked ${formatDistance(
+                                              parseISO(data.site.updateCheck.checkedAt),
+                                              new Date(),
+                                              {
+                                                  addSuffix: true,
+                                              }
+                                          )}`
+                                        : 'Never checked for updates'}
+                                </span>
+                            </small>
+                        )}
+                    </div>
+                </>
+            )}
+        </Container>
+
+        <small>
+            {autoUpdateCheckingEnabled
+                ? 'Automatically checking for updates.'
+                : 'Automatic checking for updates disabled.'}{' '}
+            Change <Code>update.channel</Code> in <Link to="/site-admin/configuration">site configuration</Link> to{' '}
+            {autoUpdateCheckingEnabled ? 'disable' : 'enable'} automatic checking.
+        </small>
+    </>
+)
+
+interface SiteUpgradeReadinessProps {
+    data: SiteUpgradeReadinessResult | undefined
+    loading: boolean
+    error: ApolloError | undefined
+}
+
+const SiteUpgradeReadiness: FunctionComponent<React.PropsWithChildren<SiteUpgradeReadinessProps>> = ({
+    data,
+    loading,
+    error,
+}) => {
+    const [isExpanded, setIsExpanded] = useState(true)
+    return (
+        <>
+            {' '}
+            <PageHeader path={[{ text: 'Upgrade readiness' }]} headingElement="h2" className="mb-3 mt-3" />
+            <Container className="mb-2">
```

Comment:
I think we usually use mb-3 at the end of the page

```suggestion
            <Container className="mb-3">
```